### DISCUSSION TITLE
Regarding Bash Shell scripting

### DISCUSSION THREAD

**Discussion by pra-y** [BretFisher/ama#224] [Q&A]
As I'm working as a devops engineer in an organization I should need to know about Shell scripting so could you suggest me provide the list of questions and learning things from that I bcom expert in Bash shell scripting....
I know the basic concepts like loops conditional statement and othr stuff but I didn't do hands-on ...Could you provide me the list of questions from which I can practise.

**Comment by BretFisher** [comment]
Sorry, I don't have any of those resources. I have seen shell scripting courses on Udemy but haven't taken any.
